lnwallet/chanfunding/coin_select.go

```diff
@@ -373,7 +373,7 @@ func CoinSelectUpToAmount(feeRate chainfee.SatPerKWeight, minAmount, maxAmount,
 	)
 
 	var errInsufficientFunds *ErrInsufficientFunds
-	if err == nil { //nolint:gocritic,ifElseChain
+	if err == nil { //nolint:gocritic
```

Comment:
We could also just turn this into a `switch` statement which might allow us to remove this remaining nolint directive too?

Reply:
great point